### PR TITLE
{bio}[GCCcore/8.2.0] fastqsplitter v1.2.0 w/ Python 3.7.2

### PR DESCRIPTION
**Pull request by bjmoran**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Hi @bjmoran and welcome to EasyBuild.

The unit test complaint is that for a Python package we expect `use_pip` to be set. See 

https://easybuild.readthedocs.io/en/latest/version-specific/generic_easyblocks.html#pythonbundle

**Comment by bjmoran** [comment]
> Hi @bjmoran and welcome to EasyBuild.
> 
> The unit test complaint is that for a Python package we expect `use_pip` to be set. See
> 
> https://easybuild.readthedocs.io/en/latest/version-specific/generic_easyblocks.html#pythonbundle

Hi @verdurin, I've set `use_pip` and the tests appear to pass now. Thanks for explaining that!

File: easybuild/easyconfigs/f/fastqsplitter/fastqsplitter-1.2.0-GCCcore-8.2.0-Python-3.7.2.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+dependencies = [('Python', '3.7.2')]
+
+use_pip = True
+
+exts_default_options = {
+    'download_dep_fail': True,
+    'source_urls': [PYPI_SOURCE],
+    'sanity_pip_check': True,
+}
```
Comment (smoors) [inline easybuild/easyconfigs/f/fastqsplitter/fastqsplitter-1.2.0-GCCcore-8.2.0-Python-3.7.2.eb:26]: @bjmoran with the `PythonBundle` easyblock, download_dep_fail is already True by default, and sanity_pip_check can be used as a normal parameter (below the exts_list):
```suggestion
exts_default_options = {'source_urls': [PYPI_SOURCE]}
```

File: easybuild/easyconfigs/f/fastqsplitter/fastqsplitter-1.2.0-GCCcore-8.2.0-Python-3.7.2.eb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    (name, version, {
+        'checksums': ['14cfb45eabe00de29886dbf2ad8dedd1b9990cb82ee194b5c41291533f3b879d'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': ['bin/%(name)s'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
```
Comment (smoors) [inline easybuild/easyconfigs/f/fastqsplitter/fastqsplitter-1.2.0-GCCcore-8.2.0-Python-3.7.2.eb:41]:
```suggestion

sanity_pip_check = True

```

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3401.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/7ca804e5093263530cca5371b16fd932 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3123.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/c82b964215dda3614dac78ec8e7aed75 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2428.golett.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/c73f779b4b8cd9de36393d94464fa908 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 8.1.1911, x86_64, Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz (haswell), Python 3.6.8
See https://gist.github.com/3548200461373c1bf3790488ee35f77b for a full test report.

**Comment by smoors** [comment]
Going in, thanks @bjmoran!

**Pull request merged**